### ISSUE TITLE
Export/Import ballots

### ISSUE THREAD

**Issue by meatballs** [electora-vote/electora-app#35]
We should offer a way to export and import the list of ballots so that users can use multiple devices without any login.

Perhaps a QR code that contains a list of ballot ids?

**Comment by theref** [comment]
by users, you mean people who are creating elections, rather those who are voting? i don't think there's any issue with forcing those people to login and maintaining a simple mapping of election id to creator

**Comment by meatballs** [comment]
No, I mean any user. If the list of ballots I have on my desktop machine is several I've created and a few more I've voted in, I'd like a simple way to get that same list on my phone without needing an account with us or anyone else.
